kubernetes/istio-auth.yaml

```diff
@@ -0,0 +1,343 @@
+# GENERATED FILE. MUTUAL_TLS, RBAC and auth enabled - requires 1.6
+# TO UPDATE, modify files in istio-install and run updateVersion.sh
+---
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: istio-ca
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        istio: istio-ca
+    spec:
+      containers:
+      - name: istio-ca
+        image: docker.io/istio/istio-ca:2017-04-13-17.43.08
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: NAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+---
+---
```

Comment:
Fixed - for consistency all 'source' files in istio-install have "---" at the end, and don't have it at startup.